netbout-web/src/test/java/com/netbout/rest/bout/TkStartTest.java

```diff
@@ -58,6 +60,9 @@
  *  should be added, e.g. when a friend is not found, the error
  *  should be reported correctly and bout creation should be
  *  discarded.
+ * @todo #750:30min TkStart should not create new bout
+ *  when same token is specified. Currently set the test case
```

Comment:
@kitsook you should mention which method should be unignored when this todo is done